### PR TITLE
solutions and programs url added

### PR DESCRIPTION
**Pull request by praveenKDass**

### CODE REVIEW

File: elevate-survey/constants/routes.js [hunk 1]
```diff
@@ -70,6 +80,236 @@ module.exports = {
...
+			sourceRoute: '/samiksha/v1/solutions/addRolesInScope/:id',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/samiksha/v1/solutions/addRolesInScope/:id',
+				type: 'POST'
+			},
+		},
+		{
```
Comment (VISHNUDAS-tunerlabs) [inline elevate-survey/constants/routes.js:203]: @praveenKDass Please check if you have to add one more config for all endpoints where we pass /:id such that the other configuration doesn't have :id
eg : /samiksha/v1/solutions/addRolesInScope

File: elevate-survey/constants/routes.js [hunk 2]
```diff
@@ -70,6 +80,236 @@ module.exports = {
 				type: 'GET'
 			},
 		},
+		{
```
Comment (VISHNUDAS-tunerlabs) [inline elevate-survey/constants/routes.js:83]: @praveenKDass make sure all api calls which accept request body their method is post

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]

File: elevate-survey/constants/routes.js (hunk 1 above)
Comment (praveenKDass) [reply elevate-survey/constants/routes.js:293]: Config all the endpoits which has :id @VISHNUDAS-tunerlabs

File: elevate-survey/constants/routes.js (hunk 2 above)
Comment (praveenKDass) [reply elevate-survey/constants/routes.js:83]: handled api methods @VISHNUDAS-tunerlabs

**Pull request merged**